src/reducers/entities/channels.ts

```diff
@@ -524,6 +524,40 @@ function stats(state: RelationOneToOne<Channel, ChannelStats> = {}, action: Gene
 
         return state;
     }
+    case ChannelTypes.INCREMENT_PINNED_POST_COUNT: {
+        const nextState = {...state};
+        const id = action.id;
+        const nextStat = nextState[id];
```

Comment:
I understand the purpose of this, but it seems like a typo that's going to cause us problems down the line. Perhaps we could just refer to it as `nextState[id]` all over to avoid the weird naming?

Reply:
Oh, that's the existing wording used in the rest of the reducer. This is fine to leave as-is then since it's better to be consistent.